node-red/Dockerfile

```diff
@@ -5,11 +5,16 @@
 # build the node red image using the offical node red distribution
 FROM nodered/node-red-docker:0.16.2
```

Comment:
should this be versioned?

Reply:
yes. ( As replied in previous discussion ).